percona-packaging/scripts/psmdb_builder.sh

```diff
@@ -139,6 +134,15 @@ get_sources(){
         git clean -xdf
         git checkout "$BRANCH"
     fi
+
+    if [ -z "${PSM_VER}" ]; then
+        PSM_VER=$(git describe --tags | sed 's/^psmdb-//' | sed 's/^r//' | awk -F '-' '{print $1}')
```

Comment:
I think this works correctly with psmdb-1.2.3-4.5 tags and r1.2.3 tags. But when the tag is something like r4.2.0-rc4 the 'rc4' will be incorrectly treated as release version. Also MONGO_TOOLS_TAG will be assigned value without '-rc4' suffix.

Reply:
When the last tag is for example r4.0.11 the 'git describe --tags' will return something like 'r4.0.11-31270-ga05ac0259c' but 31270 is not release version. To avoid this I propose adding '--abbrev=0' parameter to the 'git describe' command.

Reply:
done